File: Previous Rewrite/CombatTracker.DomainOld/Reference/EnumDefinitions.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CombatTracker.Domain.Reference
{
    public enum CriticalLevels
    {
        Same,
        A,
        B,
        C,
        D,
        E
    }

    public enum CharacterType
    {

        NPC,
        PC
    }

    public enum ActorActionType
    {
        None,
        Normal,
        Attack,
        Spell,
        Prep,
        Movement,
        Bleed,
        Critical,
        Death,
        Psychic,
        SpellEffect
    }

    public enum ActionProblem
    {
        IsOK,
        IsWarning,
        IsError
    }

    public enum ActionTypeEnum
    {
        None,
        Current,
        Proposed,
        Next,
        Effect
    }

    public enum BaseActionType
    {
        None,
        Normal,
        Attack,
        Spell,
        Prep,
        Movement
    }

    public enum ParryType
    {
        Fine,
        Must_Parry,
        No_Parry
    }

    
}

File: CombatTracker/CombatTracker.Web/ClientApp/app/entities/reference/EnumDefinitions.ts


module entities.reference {


    export enum CriticalLevels {
        Same,
        A,
        B,
        C,
        D,
        E
    }

    export enum CharacterType {

        NPC,
        PC
    }

    export enum ActorActionType {
        None,
        Normal,
        Attack,
        Spell,
        Prep,
        Movement,
        Bleed,
        Critical,
        Death,
        Psychic,
        SpellEffect
    }

    export enum ActionProblem {
        IsOK,
        IsWarning,
        IsError
    }

    export enum ActionTypeEnum {
        None,
        Current,
        Proposed,
        Next,
        Effect
    }

    export enum BaseActionType {
        None,
        Normal,
        Attack,
        Spell,
        Prep,
        Movement
    }

    export enum ParryType {
        Fine,
        Must_Parry,
        No_Parry
    }

}